types/electron-clipboard-extended/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for electron-clipboard-extended 1.x
+// Project: https://github.com/arjun-g/electron-clipboard-extended
+// Definitions by: Jungwoo An <https://github.com/Jungwoo-An>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.3
+
+import * as Electron from 'electron';
+
+interface ClipboardExtended extends Electron.Clipboard {
+    off: (event: string) => void;
```

Comment:
The optional `listener` parameter is missing. I don't know much about the library but by looking at the examples seems to be something like this:
```suggestion
    off: (event: string, listener?: () => void) => void;
```